app/UI/app.tsx

```diff
@@ -701,3 +702,24 @@ const sync_reaction_events = async (
 export function hours_ago(time: number, hours: number) {
     return time - hours * 60 * 60;
 }
+
+const sync_space_members = async (
+    pool: ConnectionPool,
+    database: Database_View,
+) => {
+    for (const relay of pool.getRelays()) {
+        forever((async () => {
+            const chan = relay.getSpaceMembersStream();
+            for await (const spaceMembers of chan) {
+                console.log(spaceMembers, relay.url);
+                if (spaceMembers instanceof Error) {
+                    console.error(spaceMembers);
+                } else {
+                    for (const spaceMember of spaceMembers) {
+                        await database.addEvent_v2(spaceMember, new URL(relay.url));
+                    }
+                }
+            }
+        })());
+    }
+};
```

Comment:
Add space. Cannot synchronize. After refreshing the page, synchronization is possible.